recipes/pysr/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "0.7.0a1" %}
+{% set pyjulia_version = "0.5.7" %}
+
+package:
+  name: pysr
+  version: {{ version }}
+
+source:
+  - url: https://github.com/MilesCranmer/PySR/archive/v{{ version }}.tar.gz
+    sha256: f002094c18182e6b00f6e0a13452bc7f80d891623932f372e8e6aa15fdf14ce4
+    folder: pysr
+  # PyJulia is not on conda-forge, so we also build it here.
+  - url: https://files.pythonhosted.org/packages/33/90/ba856cddadd923c94b7ce07dd3623d37cfea22b815076f0c70a47d867a3e/julia-{{ pyjulia_version }}.tar.gz
```

Comment:
Instead of waiting for the other PR, you can try to add pyjulia in a second recipe here and then you will have two packages get reviewed and published simultaneously. The build system will figure out that pysr needs pyjulia and will be build it first and make it available. You should try it :)

Reply:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once